src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.3
++++++
+* Added new commands for marketplace and budget API. Update to pricesheet to include top filter
```

Comment:
This should be 2 lines since there are 2 changes. The changes should be more specific.

* Added commands: `consumption budget` and `consumption marketplace`.
* `consumption pricesheet list`: Added support for `--top` filter.

Reply:
done